diff --git a/lib/tasks/gitlab/check.rake b/lib/tasks/gitlab/check.rake
index baa706d..47730a1 100644
--- a/lib/tasks/gitlab/check.rake
+++ b/lib/tasks/gitlab/check.rake
@@ -400,9 +400,9 @@ namespace :gitlab do
       check_repo_base_permissions
       check_can_clone_gitolite_admin
       check_can_commit_to_gitolite_admin
-      check_post_receive_hook_exists
-      check_post_receive_hook_is_up_to_date
-      check_repos_post_receive_hooks_is_link
+      #check_post_receive_hook_exists
+      #check_post_receive_hook_is_up_to_date
+      #check_repos_post_receive_hooks_is_link
       check_repos_git_config
 
       finished_checking "Gitolite"
